cypress/integration/DeleteTest.ts

```diff
@@ -0,0 +1,85 @@
+describe("Delete Test", () => {
+  it("Instance", () => {
+    cy.visit("/");
+
+    cy.ouiaId("Instances list table", "PF4/Table")
+      .ouiaId("Instance ten", "PF4/TableRow")
+      .find("td")
+      .then(($cells) => {
+        expect($cells).have.length(4);
```

Comment:
To be consistent with Processor, can you check status is ready

Reply:
sure, done